pkg/util/mount/mount_linux_test.go

```diff
@@ -916,3 +917,71 @@ func TestSearchMountPoints(t *testing.T) {
 		}
 	}
 }
+
+func TestGetDiskFormat(t *testing.T) {
+	tests := []struct {
+		description    string
+		expectedFstype string
+		execScripts    []ExecArgs
+		expectedError  error
+	}{
+		{
+			description:    "Test GetDiskFormat when blkid o/p is multiple lines",
+			expectedFstype: "ext4",
+			execScripts: []ExecArgs{
+				{"blkid", []string{"-p", "-s", "TYPE", "-s", "PTTYPE", "-o", "export", "/dev/foo"}, "DEVNAME=/dev/foo\nTYPE=ext4\n", nil},
+			},
+			expectedError: nil,
+		},
+		{
+			description:    "Test GetDiskFormat when blkid fails with error",
+			expectedFstype: "",
```

Comment:
this test case is when blkid fails with exit code 2 because disk is not formatted. Maybe add this explanation into the description and have another test case with other type of failures?

Reply:
sure, will add . Thanks